### ISSUE TITLE
Security vulenarability with xmldom

### ISSUE THREAD

**Issue by simeon9696** [firebase/firebase-tools#3219]
After running `npm update` a security vulnerability was reported. Below shows the output of `npm audit`

```
                       === npm audit security report ===                        


                                 Manual Review                                  
             Some vulnerabilities require your attention to resolve             
                                                                                


  Low             Misinterpretation of malicious XML input                      

  Package         xmldom                                                        

  Patched in      >=0.5.0                                                       

  Dependency of   firebase-tools                                                

  Path            firebase-tools > plist > xmldom   
```

Running `npm audit fix` was unable to resolve this. Is this something to be concerned about?

"firebase-tools": "^9.6.1",

**Platform:**
Windows 10 Pro 20H2

**Comment by bkendall** [comment]
It looks like `plist` is going to have to update their dependencies in order for us to be able to fix this on our side. They have an issue open already (though I'm a little skeptical because it's been years since that repo has had an update): https://github.com/TooTallNate/plist.js/issues/105

**Comment by MrAndrew** [comment]
Is there at least a functional fix if the dependencies can't be updated? For example, editing the dependency code or forking the repo, fix it, and depend on that instead of the original?

**Comment by IchordeDionysos** [comment]
@bkendall is the `plist` package even used in `firebase-tools`?
 
 The only occurrence I found for `plist` was in the `package.json` and lock file and as a file ending of the `GoogleService-Info.plist`:
 
![image](https://user-images.githubusercontent.com/10195482/112455215-cd3f1000-8d59-11eb-9dcd-ccfb79b86e01.png)

So maybe it can just be removed?

**Comment by JanCVanB** [comment]
```sh
$ npm audit fix
...
    firebase-tools  >=7.3.0
    Depends on vulnerable versions of plist
    node_modules/firebase-tools
...
To address all issues (including breaking changes), run:
  npm audit fix --force
```
😯
```sh
$ npm audit fix --force
...
npm WARN audit Updating firebase-tools to 7.2.4,which is a SemVer major change.
...
          firebase-tools  3.10.0 - 7.16.2
          Depends on vulnerable versions of google-auto-auth
          node_modules/firebase-tools
...
To address all issues (including breaking changes), run:
  npm audit fix --force
```
🤔
```sh
$ npm audit fix --force
...
npm WARN audit Updating firebase-tools to 9.7.0,which is a SemVer major change.
...
    firebase-tools  >=7.3.0
    Depends on vulnerable versions of plist
    node_modules/firebase-tools
...
To address all issues (including breaking changes), run:
  npm audit fix --force
```
🤦

**Comment by bkendall** [comment]
1. This was fixed by #3248
2. You're right that `plist` isn't used. I'm not sure why it's still there. I'll make a quick PR to take care of it.
